src/proc.py

```diff
@@ -30,7 +30,8 @@
 
 
 def FixPython(cmd):
-  if cmd[0].endswith('.py'):
+  script = cmd[0]
+  if script.endswith('.py') and os.path.exists(script):
```

Comment:
This could use the same logic as `IsExe()` in the previous version of the patch that broke all of this